Added integrating with Slack topic (#34024)
openshift/openshift-docs: modules/send-alerts-to-different-slack-channels.adoc

// Module included in the following assemblies:
//
// * integration/integrate-with-slack.adoc
:_module-type: PROCEDURE
[id="send-alerts-to-different-slack-channels_{context}"]
= Sending alerts to different Slack channels

[role="_abstract"]

> You can configure {product-title} to send notifications to different Slack channels, so they directly go to the right team.

Reviewer: s/channels, so they/channels so that they
**so** _conjunction_
Do not use by itself as a conjunction. Use a less ambiguous alternative such as "so that" or "therefore".

.Procedure
. After you configure incoming webhooks, add an annotation similar to the following in your deployment YAML file:
+
[source,yaml]
----
example.com/slack-webhook: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX
----
. Use the annotation key `example.com/slack-webhook` in the *Label/Annotation Key For Slack Webhook* field when you configure {product-title}.

> Once configured, if a deployment has the annotation, {product-title} sends the alert to the Webhook URL specified in the annotation.

Reviewer: ```suggestion
After you configure the annotation for the deployment, {product-title} sends the alert to the webhook URL specified in the annotation.
```
^Not sure if that changes the meaning of your statement, but I would suggest trying to make the most important element of the statement more clear...namely, that RHACS sends the newly configured alert. 

**once** _adverb_
Use only to mean "one time". Do not use as a conjunction to mean "after" or "when".

Author: Yeah, that changes the meaning. I've wrote it again as:
```
After the configuration is complete, if a deployment has the annotation that you configured in the YAML file, {product-title} sends the alert to the Webhook URL you specified for that annotation. Otherwise, it sends the alert to the default Webhook URL.
```

> Otherwise, it sends the alert to the default Webhook URL.

Reviewer: s/Webhook/webhook
